python/bot/operator_bot.py

```diff
@@ -130,18 +132,23 @@ def respond_to_chat_request(event):  # pylint: disable=unused-variable
             return client.submit_exercise(event.cid, 'CreateChatRequestReject')
 
 
-    @client.ledger_created('DABL.Ledger.V3.LedgerParty')
+    @client.ledger_created('Chat.UserSession')
     def invite_user_to_chat(event):  # pylint: disable=unused-variable
-        logging.info(f'On DABL.Ledger.V3.LedgerParty created!')
+        logging.info(f'On Chat.UserSession created!')
         cdata = event.cdata
-        if cdata['party'] == client.party:
+        if cdata['user'] == client.party:
```

Comment:
Not sure this check is required any more, as the `UserAdmin` won't have created a `Chat.UserSession`. Would go so far as to assert that the `operator` and `user` can't be the same party in DAML just to make sure this case never happens.